packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -496,29 +495,33 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
 }
 
-const convertEventToChatMessage = (event: ChatMessageReceivedEvent): ChatMessage => {
+const convertEventToChatMessage = (
+  event: ChatMessageReceivedEvent | ChatMessageEditedEvent | ChatMessageDeletedEvent
+): ChatMessage => {
   return {
     id: event.id,
     version: event.version,
-    content: { message: event.message },
+    content: isChatMessageDeletedEvent(event) ? undefined : { message: event.message },
```

Comment:
I'm wondering if Chat DSK returns content for `messageDeleted` event and if so, if we need to be consistent here

Reply:
It explicitly doesn't have it in the payload so had to add this line here